packages/graphic-walker/src/components/limitSetting.tsx

```diff
@@ -1,25 +1,23 @@
 import React, { useMemo } from 'react';
 import { useTranslation } from 'react-i18next';
-import { useDebounceValueBind } from '../hooks';
+import { createUseDebounceValueBind } from '../hooks';
 import { Checkbox } from './ui/checkbox';
-import { Slider } from './ui/slider';
+import { Input } from './ui/input';
 
 export default function LimitSetting(props: { value: number; setValue: (v: number) => void }) {
     const { t } = useTranslation('translation', { keyPrefix: 'main.tabpanel.settings' });
-    const [innerValue, setInnerValue] = useDebounceValueBind(props.value, (v) => props.setValue(v));
-    const sliderValue = useMemo(() => (innerValue > 0 ? [innerValue] : [0]), [innerValue]);
+    const [innerValue, setInnerValue] = createUseDebounceValueBind(600)(props.value, (v) => props.setValue(v));
+    const inputValue = useMemo(() => (innerValue > 0 ? innerValue : 0), [innerValue]);
```

Comment:
input value should not be a computated value, it will leads to the problem that user cannot delete all input numbers and re-enter new numbers.